APIManager.js

```diff
@@ -0,0 +1,65 @@
+class APIManager {
+  constructor() {
+    this.data = {};
+  }
+
+  getUserInfo() {
+    $.ajax({
+      url: "https://randomuser.me/api/?results=7",
+      dataType: "json",
+      async: false,
+      success: (data) => {
```

Comment:
also use the `error` part of the `.ajax` call. It is important to start learning to error handle